commercial/app/controllers/CreativeTestPage.scala

```diff
@@ -1,14 +1,14 @@
 package commercial.controllers
 
 import conf.Configuration
-import model.{ApplicationContext, GuardianContentTypes, MetaData, SectionSummary}
+import model.{ApplicationContext, DotcomContentType, MetaData, SectionId}
```

Comment:
Renamed `GuardianContentType` into `DotcomContentType` because CAPI ones are slightly different from Dotcom ones. I wanted to avoid any confusion